velox/exec/Driver.cpp

```diff
@@ -643,6 +653,11 @@ StopReason Driver::runInternal(
                   curOperatorId_,
                   kOpMethodIsBlocked);
               if (blockingReason_ != BlockingReason::kNotBlocked) {
+                VELOX_CHECK(
```

Comment:
Can add a macro for this then? Like CHECK_IS_BLOCK_FUTRUE? Thanks!

Reply:
Added, thanks.